### PR TITLE
create ctmpl and put secrets in vault

### PR DESCRIPTION
**Pull request by jacmrob**

### CODE REVIEW

File: test/integration_test.sh [hunk 1]
```diff
@@ -232,18 +230,14 @@ elif [ $ss2_mode == "local" ]; then
...
 printf "\n\nCreating Lira config"
-printf "\nUsing $env_config_json"
-printf "\nUsing $secrets_json\n"
-python $script_dir/create_lira_config.py \
-    --env_config_file $env_config_json \
-    --secrets_file $secrets_json \
-    --tenx_prefix $tenx_prefix \
-    --ss2_prefix $ss2_prefix \
-    --pipeline_tools_prefix $pipeline_tools_prefix > config.json
+docker run -it --rm -v $PWD:/working -e VAULT_TOKEN=$vault_token \
```
Comment (dshiga) [inline test/integration_test.sh]: Getting an error here. I was running from a subdirectory that I made: secondary-analysis/test/test-runs. I got farther by changing this to ```docker run -it --rm -v $repo_dir:/working -v $work_dir:/out -e VAULT_TOKEN=$vault_token \``` (I also set repo_dir in the script to be the root of the secondary-analysis repo) and then below setting ```OUT_PATH=/out```. But I then ran into some vault auth errors which are probably due to something I'm doing wrong.

File: test/integration_test.sh [hunk 2]
```diff
@@ -232,18 +230,14 @@ elif [ $ss2_mode == "local" ]; then
 fi
 
 # 6. Create config.json
-# (TODO: Use Henry's script here)
```
Comment (dshiga) [inline test/integration_test.sh]: Comment "Create config.json" above should also change to reflect the new file name.

**Comment by dshiga** [comment]
Thinking this through a little bit, I'd really like to restrict the role of vault as much as possible. The rationale for this is that we want Lira to be easy for others to configure and use outside of Broad and HCA for other projects, even if they don't use vault.

Could we go back to something closer to what we had before, where we had a secrets json file, an environment config json file, and a script to generate the final Lira config from the two of them? Vault would only render the secrets json file in that case, and anyone not using Vault could more easily replace that step with their own method. I think this would also make this script a bit easier for mint devs to use, since we wouldn't necessarily have to use vault every time.

**Comment by jacmrob** [comment]
okay that's fair.  in that case, do we still want to use ctmpl for the env-specific rendering of the configs rather than that python script?

**Pull request closed without merging**